cypress/e2e/buyProductsAsExistingCustomer.cy.ts

```diff
@@ -17,11 +25,8 @@ describe("Buy product as existing user", () => {
     cy.addAliasToGraphRequest("user")
       .addAliasToGraphRequest("checkoutShippingAddressUpdate")
       .addAliasToGraphRequest("checkoutBillingAddressUpdate")
-      .addAliasToGraphRequest("checkoutDeliveryMethodUpdate")
-      .get(SHARED_ELEMENTS.productsList)
-      .children()
-      .first()
-      .click();
+      .addAliasToGraphRequest("checkoutDeliveryMethodUpdate");
```

Comment:
Try to keep adding aliases on the very begining of every 'it'. If not alias could not be created fast enough and wait will fail